src/Symfony/Component/Form/Resources/translations/validators.sv.xlf

```diff
@@ -14,6 +14,126 @@
+                <target>Välj ett giltigt datumintervall.</target>
+            </trans-unit>
+            <trans-unit id="107">
+                <source>Please enter a valid date and time.</source>
+                <target>Ange datum och tid.</target>
+            </trans-unit>
+            <trans-unit id="108">
+                <source>Please enter a valid date.</source>
+                <target>Ange ett datum.</target>
+            </trans-unit>
+            <trans-unit id="109">
+                <source>Please select a valid file.</source>
+                <target>Välj en fil.</target>
+            </trans-unit>
+            <trans-unit id="110">
+                <source>The hidden field is invalid.</source>
+                <target>Det dolda fältet är ogiltigt.</target>
+            </trans-unit>
+            <trans-unit id="111">
+                <source>Please enter an integer.</source>
+                <target>Ange ett heltal.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>Välj språk.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>Välj plats.</target>
+            </trans-unit>
+            <trans-unit id="114">
+                <source>Please enter a valid money amount.</source>
+                <target>Ange en giltig summa pengar.</target>
+            </trans-unit>
+            <trans-unit id="115">
+                <source>Please enter a number.</source>
+                <target>Ange en siffra.</target>
+            </trans-unit>
+            <trans-unit id="116">
+                <source>The password is invalid.</source>
+                <target>Lösenordet är ogiltigt.</target>
+            </trans-unit>
+            <trans-unit id="117">
+                <source>Please enter a percentage value.</source>
+                <target>Ange ett procentuellt värde.</target>
+            </trans-unit>
+            <trans-unit id="118">
+                <source>The values do not match.</source>
+                <target>De angivna värdena stämmer inte överens.</target>
+            </trans-unit>
+            <trans-unit id="119">
+                <source>Please enter a valid time.</source>
+                <target>Ange en giltig tid.</target>
+            </trans-unit>
+            <trans-unit id="120">
+                <source>Please select a valid timezone.</source>
+                <target>Välj tidszon.</target>
+            </trans-unit>
+            <trans-unit id="121">
+                <source>Please enter a valid URL.</source>
+                <target>Ange en giltig URL.</target>
+            </trans-unit>
+            <trans-unit id="122">
+                <source>Please enter a valid search term.</source>
+                <target>Ange ett giltigt sök-begrepp.</target>
```

Comment:
What do you think about this?

```suggestion
                <target>Ange en giltig sökterm.</target>
```

Reply:
I'd say "sökterm" is an translated "anglocism" from search term but I would argue "sökbegrepp" is a more proper translation. But at the same time, I would expect a regular user to understand both equally well.

Reply:
Okey. Cool cool. 

Just make sure to make it "sökbegrepp" instead of "sök-begrepp".

Reply:
Yes, good point, it is adjusted now, thanks!